docs/assemblies/custom_resources.adoc

```diff
@@ -319,11 +319,6 @@ OpenStackDataPlaneServiceSpec defines the desired state of OpenStackDataPlaneSer
 | map[string]<<openstackdataplaneservicecert,OpenstackDataPlaneServiceCert>>
 | false
 
-| play
-| Play is an inline playbook contents that ansible will run on execution.
-| string
-| false
-
```

Comment:
I spent a couple of minutes looking into why the new field doesn't show up here. And afaict, it looks like there might be a bug in crd-to-markdown in how it's looking for the "inline" string. Because if that string is part of the field name, it won't render the field.

we  might need to change the name of the field to work around this. PlaybookContents perhaps.